### PR TITLE
Adds Twitter social-wall as a loklak app

### PR DESCRIPTION
**Pull request by sopu**
Adds twitter social-wall to the set of loklak apps

### CODE REVIEW

**Comment by sopu** [comment]
@Orbiter Please suggest modifications if any!

File: html/apps/social-wall/index.html [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="content-type" content="text/html; charset=utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <meta name="Content-Language" content="en, english"/>
+    <meta name="author" content="@sopu"> 
+    <link rel="icon" type="image/png" href="/artwork/favicon.png">
+    <title>Masonry Layout with Masonry Plugin</title>
```
Comment (sudheesh001) [inline html/apps/social-wall/index.html]: Please rename this to `Loklak wall - Masonry Layout`

File: html/apps/social-wall/js/social_wall.js [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+{
+  var base_url = 'http://twitter.com/';   // identica: 'http://identi.ca/'
+  var hashtag_part = 'search?q=%23';        // identica: 'tag/'
+  // convert URLs into links
+  /*text = text.replace(
+      /(>|<a[^<>]+href=['"])?(https?:\/\/([-a-z0-9]+\.)+[a-z]{2,5}(\/[-a-z0-9!#()\/?&.,]*[^ !#?().,])?)/gi,
+      function($0, $1, $2) {
+          return ($1 ? $0 : '<a href="' + $2 + '" target="_blank">' + $2 + '</a>');
+      });*/
+  // convert protocol-less URLs into links
```
Comment (sudheesh001) [inline html/apps/social-wall/js/social_wall.js]: remove unused comment code.
```
-  // convert URLs into links
-  /*text = text.replace(
-      /(>|<a[^<>]+href=['"])?(https?:\/\/([-a-z0-9]+\.)+[a-z]{2,5}(\/[-a-z0-9!#()\/?&.,]*[^ !#?().,])?)/gi,
-     function($0, $1, $2) {
-          return ($1 ? $0 : '<a href="' + $2 + '" target="_blank">' + $2 + '</a>');
-      });*/
-  // convert protocol-less URLs into links  
```

File: html/apps/social-wall/js/social_wall.js [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+    $(".card").on("click", function(){
+      window.open($(this).attr('data-link'), '_blank');
+    });
+
+    $(".card__author-content").on("click", function(){
+      window.open($(this).attr('data-author-link'), '_blank');
+    });
+
+  });
+});
```
Comment (sudheesh001) [inline html/apps/social-wall/js/social_wall.js]: Add new line.

**Comment by sudheesh001** [comment]
Looks good @sopu Please make the required corrections,

**Pull request closed without merging**